packs/book-of-the-dead-bestiary/gashadokuro.json

```diff
@@ -191,7 +191,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The gashadokuro breathes a spray of bone shards in a @Template[cone|distance:30]. Each creature in the area takes @Damage[8d12[piercing]] damage (@Check[reflex|dc:34|basic]).</p>\n<p>It can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>"
+                    "value": "<p>The gashadokuro breathes a spray of bone shards in a @Template[cone|distance:30]. Each creature in the area takes @Damage[8d12[piercing]|options:area-damage]] damage (@Check[reflex|dc:34|basic|options:area-effect]).</p>\n<p>It can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>"
```

Comment:
```suggestion
                    "value": "<p>The gashadokuro breathes a spray of bone shards in a @Template[cone|distance:30]. Each creature in the area takes @Damage[8d12[piercing]|options:area-damage] damage (@Check[reflex|dc:34|basic|options:area-effect]).</p>\n<p>It can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>"
```